test/ServiceFileGenerator.spec.ts

```diff
@@ -0,0 +1,41 @@
+import 'mocha';
+import { expect } from "chai";
+
+import Logger from "../src/utils/Logger";
+Logger.LogToConsole = false;
+
+import ServiceFileGenerator from "../src/codegen/generators/ServiceFileGenerator";
+import ServiceNode from "../src/codegen/ir/ServiceNode";
+import FileCreator from '../src/utils/FileCreator';
+
+describe("ActionFileGenerator",  () => {
+    it("Produces the correct output for three actions", () => {
+        let node = new ServiceNode('users');
+        node.addServiceCase({type: 'GET_ALL_USERS', method: "GET", url: '/api/users'});
+        node.addServiceCase({type: 'UPDATE_ALL_USERS', method: "POST", url: '/api/users'});
+        let code = new ServiceFileGenerator(node).codeGen();
+        // FileCreator(code, './service.js')
+        expect(code).to.equal('export const getAllUsers = () => {\n' +
```

Comment:
Is this the best way to make these expectations? Wouldn't using 
```
` ... my expected codegen here with beautiful tabs ... ` 
```
also work?

Reply:
Eh I mean, it could, its just a pain to get it to be formatted exactly correctly that way. This functions more as a regression test as it is anyway. I don't think its a big deal